modules/ROOT/partials/integrations/svelte-quick-start.adoc

```diff
@@ -38,99 +43,127 @@ npm install @tinymce/tinymce-svelte
 endif::[]
 ifeval::["{productSource}" == "cloud"]
 
-. Open `+src/App.svelte+` and add:
-* An `+import+` statement for the {productname} component.
-* The `+<Editor />+` as a placeholder for the editor.
+. Open `+src/App.svelte+` and replace the contents with:
 +
-For example:
-+
-_File:_ `+src/App.svelte+`
-+
-[source,html]
+[source,html,subs="+macros"]
 ----
-<script lang="ts">
+<script>
```

Comment:
Why remove `lang="ts"`?

Does `npm create vite@5 tinymce-svelte-demo -- --template svelte` not add Typescript support into the created project or something?

Reply:
Nvm, saw your comment on the other PR